SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -34,6 +34,8 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
                 "OnlineSubsystemUtils",
                 "InputCore",
                 "Sockets",
+                "GameplayAbilities",
```

Comment:
Am I right in thinking these are no longer needed as of the latest commit?

Reply:
👍👍👍